developer/coding-guide.md

```diff
@@ -0,0 +1,83 @@
+# Coding style guide
+
+A guide to working on code at Reaction, in bite-size steps!
+
+## Git: Issues, branches, commits
+
+Git version control is how we work together as a team. Naming branches and writing commit messages helps us keep a easy-to-understand history of the changes in the project.
+
+Follow these guidelines to make releases and code reviews smoother.
+
+1. Find a GitHub issue for what you're working on.
+
+2. Branch name: Create a branch name, with the ID number of the GitHub issue, in the following style:
+`yourhandle-type-###-slug`
+
+```
+spencern-feat-222-add-email
```

Comment:
Isn't this the old format? Am I looking at this wrong?

Reply:
That's what I was thinking. Also not sure what happened to our old comments, but if you go back to the first commit in this PR, I thought we'd decided that it was going to be
`type-###-name-slug`
or potentially
`type-###-handle-slug` if we don't want to get confusing by using real names in place of handles